src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/DeliusWebClientConfiguration.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.config
+
+import org.springframework.context.ApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager
+import org.springframework.security.oauth2.client.registration.ClientRegistration
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClient.Builder
+import uk.gov.justice.digital.hmpps.manageusersapi.adapter.WebClientUtils
+
+@Configuration
+class DeliusWebClientConfiguration(appContext: ApplicationContext) :
+  AbstractWebClientConfiguration(appContext, "delius") {
+
+  @Bean("deliusClientRegistration")
+  fun getDeliusClientRegistration(): ClientRegistration = getClientRegistration()
+
```

Comment:
This was needed otherwise the webclient doesn't get recognised and puts the client on a random port